### ISSUE TITLE
Todos should persist when the page is refreshed

### ISSUE THREAD

**Issue by toxoplasmotic** [SeanECogan/todo-it#2]
I refreshed, and... all of my precious todos are gone!

Maybe use local storage to cache the list? You could pull in [angular-local-storage](https://www.npmjs.com/package/angular-local-storage) to help with that.

[If you want to be fancy](https://github.com/firebase/angularfire), cache the list in [Firebase](https://firebase.google.com)'s realtime database. A Firebase/Angular todo list has been [done before](http://todomvc.com/examples/firebase-angular), though. :)

**Comment by SeanECogan** [comment]
Yeah, I was thinking of using local storage, or something like that. It's definitely something I'm planning on adding!
